<launch>
<node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen" ns="/camera">

    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="fps" type="int" value="10" />
    <param name="frame" type="string" value="camera" />
   <param name="auto_focus" type="bool" value="False" />
   <param name="focus_absolute" type="int" value="0" />
   <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->

   <param name="device" type="string" value="/dev/video0" />
  <!-- <param name="camera_info_url" type="string" value="file://$(find locateCamera)/src/ost.yaml" />-->

    </node>
        <arg name="marker_size" default="2.471" />
        <arg name="max_new_marker_error" default="0.08" />
        <arg name="max_track_error" default="0.2" />

        <arg name="cam_image_topic" default="/limited" />
        <arg name="cam_info_topic" default="/camera/camera_info" />

        <arg name="output_frame" value="/camera" />
       <!-- <arg name="bundle_files" default="/home/josh/catkin_ws/ARPannel.xml" />-->

<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect"  respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) " />
	<node name= "rviz" pkg = "rviz" type = "rviz" />
        <node name= "limiter" pkg = "locationCamera" type = "cameraLimiter" />
</launch>
